PythonVisualizations/SimpleSorting.py

```diff
@@ -205,34 +202,34 @@ def insert(self, val):
         self.moveItemsTo(cellPair, toPositions, steps=self.CELL_SIZE, sleepTime=0.01)
 
         # add a new Drawable with the new value, color, and display objects
-        self.list.append(drawable(
-            val, self.canvas.itemconfigure(cellPair[0], 'fill'), *cellPair))
+        self.list.append(drawable(val, self.canvas.itemconfigure(cellPair[0], 'fill'), *cellPair))

```

Comment:
This was a bug in the former version, but might as well be fixed here.  To get the fill color from the call to `self.canvas.itemconfigure()`, you have to extract the last element of the tuple.  It should be `self.canvas.itemconfigure(cellPair[0], 'fill')[-1]` instead